rpcserver.go

```diff
@@ -2936,6 +2939,174 @@ func (s *rpcServer) determineAccountVersion(
 	return account.VersionInitialNoVersion, nil
 }
 
+type ActionFee struct {
+	Action  string
+	BlockHeight int32
+	TimeStamp int64
```

Comment:
nit: we usually do `Timestamp` and only use uppercase when a new word is starting

Reply:
Will do. I think I was inspired by https://github.com/lightningnetwork/lnd/blob/2e410dd248a7de1beee4e305d510d5b667bb24f5/lnrpc/lightning.pb.go#L1708